components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1997,7 +1997,24 @@ do_build_config() {
 do_default_build_config() {
   build_line "Writing configuration"
   if [[ -d "$PLAN_CONTEXT/config" ]]; then
-    cp -r "$PLAN_CONTEXT/config" $pkg_prefix
+    if [ -z "${HAB_CONFIG_EXCLUDE:-}" ]; then
+      # HAB_CONFIG_EXCLUDE not set, use defaults
+      config_exclude_exts=("*.sw*" "*~" "*.bak")
```

Comment:
I think *.sw* is too broad as a default, given that it will match anything with an .sw* extension. If we're trying to match .swp and .swap, I would rather we be explicit about it.